### PR TITLE
Feature/react native snap carousel

### LINKED ISSUE #2: Automate new package creation
Use lerna init to create a new repo in this monorepo via npm script.

We'll use this to centralize all binding that are across multiple apps.

### PR DESCRIPTION
**Pull request by jrquadros**
closes #2

### CODE REVIEW

File: universe.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  {
+    "title": "bs-graphql",
+    "url": "https://github.com/Astrocoders/bs-universe/tree/master/bs-graphql"
+  },
+  {
+    "title": "react-native-snap-carousel",
+    "url": "https://github.com/Astrocoders/bs-universe/tree/master/react-native-snap-carousel"
+  }
+]
```
Comment (JeffersonCarvalh0) [inline universe.json]: There is a circleci job to automate the task of updating this file. It is triggered when there is a merge to the master branch, so I think it is not necessary to update it now

Comment (jrquadros) [reply universe.json]: Cool!

File: react-native-snap-carousel/src/SnapCarousel.bs.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// Generated by BUCKLESCRIPT VERSION 5.0.6, PLEASE EDIT WITH CARE
```
Comment (medson10) [inline react-native-snap-carousel/src/SnapCarousel.bs.js]: the bs.js files should be ignored no? @fakenickels

Comment (fakenickels) [reply react-native-snap-carousel/src/SnapCarousel.bs.js]: Yes they should!

File: universe.json (hunk 1 above)
Comment (fakenickels) [reply universe.json]: Nice talking about it we can move this CI logic to GitHub actions

**Comment by medson10** [comment]
you added the file type on ignore but you still have to remove the file from git, you can do this with `git rm src/SnapCarousel.bs.js`

File: react-native-snap-carousel/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Basic Reason Template
```
Comment (fakenickels) [inline react-native-snap-carousel/README.md]: actually the template we need to have is to have an `Installation` and then the building instructions

File: lerna.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "packages": [
-    "packages/*"
```
[the remark this answers is not in the document]
Comment (jrquadros) [reply lerna.json]: this is the standard path of lerna

**Pull request merged**